### PR TITLE
send failed test to system-test channel

### PR DESCRIPTION
**Pull request by acidsennui**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -189,6 +189,10 @@ references:
           HOOK_DATA=$(printf "$HOOK_TEMPLATE" "${CIRCLE_JOB:?}" "${CIRCLE_BRANCH:?}" "${CIRCLE_USERNAME:-unknown}" "${CIRCLE_BUILD_URL:?}")
           curl -X POST -H 'Content-Type: application/json' ${ROCKET_HOOK_URL:?} --data "${HOOK_DATA:?}"
         fi
+        if [ -n "$CIRCLE_BRANCH" -a "$CIRCLE_BRANCH" = "${NOTIFY_BRANCH:?}" ]; then
+          HOOK_DATA=$(printf "$HOOK_TEMPLATE" "${CIRCLE_JOB:?}" "${CIRCLE_BRANCH:?}" "${CIRCLE_USERNAME:-unknown}" "${CIRCLE_BUILD_URL:?}")
+          curl -X POST -H 'Content-Type: application/json' ${ROCKET_HOOK_URL_SYSTEM_TEST:?} --data "${HOOK_DATA:?}"
```
Comment (lucafavatella) [inline .circleci/config.yml]: Where is ROCKET_HOOK_URL_SYSTEM_TEST defined?

File: .circleci/config.yml [hunk 2]
```diff
@@ -189,6 +189,10 @@ references:
           HOOK_DATA=$(printf "$HOOK_TEMPLATE" "${CIRCLE_JOB:?}" "${CIRCLE_BRANCH:?}" "${CIRCLE_USERNAME:-unknown}" "${CIRCLE_BUILD_URL:?}")
           curl -X POST -H 'Content-Type: application/json' ${ROCKET_HOOK_URL:?} --data "${HOOK_DATA:?}"
         fi
+        if [ -n "$CIRCLE_BRANCH" -a "$CIRCLE_BRANCH" = "${NOTIFY_BRANCH:?}" ]; then
```
Comment (lucafavatella) [inline .circleci/config.yml]: Would this not post failure from any job in system test room?

File: .circleci/config.yml (hunk 1 above)
Comment (michalbagrowski) [reply .circleci/config.yml]: in circleCI

File: .circleci/config.yml (hunk 2 above)
Comment (michalbagrowski) [reply .circleci/config.yml]: yes, fixed

**Comment by acidsennui** [comment]
@lucafavatella fixed to send them only from system-test

File: .circleci/config.yml [hunk 3]
```diff
@@ -191,6 +191,29 @@ references:
...
       when: on_fail
 
+  fail_notification_system_test: &fail_notification_system_test
+    run:
+      name: Send failure notification on system test
+      environment:
+        NOTIFY_BRANCH: master
+        HOOK_TEMPLATE: |
+          {
+            "text": "CircleCI job **%s** failed on branch **%s** by @%s",
```
Comment (lucafavatella) [inline .circleci/config.yml]: @ThomasArts Did you want `@here` here?

**Pull request merged**